### PR TITLE
[content-service] Use buf to generate protos

### PR DESCRIPTION
**Pull request by easyCZ**
## Description

Uses buf to define configuration for proto generation. Generates identical output.

Buf makes it a bit easier to understand what is going with the protoc 
## Related Issue(s)

Fixes #

## How to test

```
cd components/content-service-api
./generate.sh
```

## Release Notes

```release-note
NONE
```

## Werft options:

- [ ] /werft with-local-preview
      If enabled this will build `install/preview`
- [ ] /werft with-preview
- [ ] /werft with-integration-tests=all
      Valid options are `all`, `workspace`, `webapp`, `ide`

### CODE REVIEW

**Review by aledbf — APPROVED** [review]

File: components/content-service-api/go/blobs.pb.go [hunk 1]
```diff
@@ -5,7 +5,7 @@
 // Code generated by protoc-gen-go. DO NOT EDIT.
 // versions:
 // 	protoc-gen-go v1.28.1
-// 	protoc        v3.20.1
+// 	protoc        (unknown)
```
Comment (jenting) [inline components/content-service-api/go/blobs.pb.go:8]: Could we make the protoc version not to unknow?

Comment (easyCZ) [reply components/content-service-api/go/blobs.pb.go:8]: Unsure. It's using the same already installed protoc, it's weird it's not reporting it into here. Will TAL.

Comment (easyCZ) [reply components/content-service-api/go/blobs.pb.go:8]: https://github.com/bufbuild/buf/issues/799#issuecomment-996881066

**Review by geropl — APPROVED** [review]
LGTM, thx @easyCZ ! :pray:

**Pull request merged**